src/supermarket/config/initializers/session_store.rb

```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_supermarket_session'
+Rails.application.config.session_store :cookie_store, key: '_supermarket_session', secure: Rails.env.production?
```

Comment:
Sadly, not everyone running a "production" Supermarket instance is using HTTPS. 😢

This check should probably be something like `ENV['PROTOCOL'] == 'https'`. That's the environment variable that is set for the Rails process to know what URL scheme to use when creating links back to itself. The value can be either `"https"` ([default](https://github.com/chef/supermarket/blob/5911828fd3b5be367028c147e7962b520782b001/omnibus/cookbooks/omnibus-supermarket/attributes/default.rb#L333)) or `"http"` for Supermarket omnibus installs, all of which operate with `Rails.env.production? == true`.